uint32 id
string name
string type
uint32 floor
geometry_msgs/Pose pose
